pkg/lib/cockpit-components-modifications.jsx

```diff
@@ -37,16 +37,33 @@ class ModificationsExportDialog extends React.Component {
         super(props);
 
         this.state = {
-            active_tab: "shell"
+            active_tab: "shell",
+            copied: false
         };
 
         this.handleSelect = this.handleSelect.bind(this);
+        this.getScript = this.getScript.bind(this);
     }
 
     handleSelect(active_tab) {
         this.setState({ active_tab });
     }
 
+    getScript() {
```

Comment:
This name is a bit confusing , copyToClipboard() would be a bit clearer. But not a biggie if that's the only outstanding thing.